deployment/terraform/modules/docker_host/main.tf

```diff
@@ -0,0 +1,57 @@
+variable "cidr_block" {
+  type = "string"
+  default = "10.0.0.0/16"
+}
+
+variable "security_group_id" {
+  type = "string"
+  default = "my_security_group_id"
+}
+
+variable "security_group_name" {
+  type = "string"
+  default = "allow_docker"
```

Comment:
I'd simply assign a value in the corresponding `module` section of `live/main.tf` based on `${terraform.workspace}` ([example](https://github.com/refinery-platform/refinery-platform/blob/ff1167230df226f27f932f846452cdbd8f901dd0/deployment/terraform/live/main.tf#L25))

Reply:
but both of those examples still have a corresponding definition in vars: for example, cognito/vars.tf:
```
variable "iam_resource_name_prefix" {
  description = "Name prefix of the S3 upload role"
}
```

I think I have what I want: This particular variable is called out as something that might be changed, but the default is fine.